src/__tests__/integration/oauth.test.ts

```diff
@@ -315,13 +315,13 @@ async function makeInstalledRequest(
     expect(response.status).toBe(200);
     expect(mock).toHaveBeenCalledTimes(1);
   } else {
-    expect(response.status).toBe(500);
+    expect(response.status).toBe(config.embedded ? 500 : 302);
     expect(mock).not.toHaveBeenCalled();
   }
 }
 
 // Fires a valid request to check that the authenticated middleware allows requests through
-async function makeAuthenticatedRequest(
+async function makevalidateSession(
```

Comment:
I don't think the word `make` is necessary and `validSession` is enough:

```suggestion
async function validSession(
```